frappe/public/js/frappe/form/form.js

```diff
@@ -292,13 +292,19 @@ frappe.ui.form.Form = class FrappeForm {
 		for (const d of data) {
 			for (const condition of d) {
 				let [doctype, field, operator, value] = condition;
+				if (value.includes("eval")) {
```

Comment:
Let's do starts with `eval:` that's more explicit? `eval` can appear inside normal words.